samples/algorithms/simple-grover/README.md

```diff
@@ -1,28 +1,46 @@
-﻿# Grover Search Sample #
+﻿---
+page_type: sample
+languages: [qsharp, python, csharp]
+products: [qdk]
+---
 
-This sample implements Grover's search algorithm. Oracles implementing the database are explicitly constructed
-together with all steps of the algorithm. See the [DatabaseSearch](../DatabaseSearch/README.md) sample for and
-extended version and the [Grover Search Kata](https://github.com/microsoft/QuantumKatas/tree/master/GroversAlgorithm)
-to learn more about Grover's algorithm and how to implement it in Q#.
+# Searching with Grover's Algorithm #
```

Comment:
It would be helpful to be more concrete somewhere regarding what to expect when it is run. I had to look at the doc comment in the code so figure out what state was even marked and that the marked state is in fact hard coded and not configurable.